### DISCUSSION TITLE
GET character values in various encodings

### DISCUSSION THREAD

**Discussion by james-eda** [cdisc-org/DataExchange-DatasetJson-API#15] [User Stories]
As a statistical programmer,  I want to migrate data from one common encoding like LATIN1 to another like UTF-8 so that I can apply one set of transformation across multiple studies

**Comment by lexjansen** [comment]
Do realize that JSON - and thus Dataset-JSON - is Unicode by definition.

**Comment by JozefAerts** [comment]
We need to check whether Latin-1 is a subset of Unicode/UTF-8.
I remember some problems with skew quotes (Windows / Excel) that gave problems when generating UTF-8 encoded ODM for example.

**Comment by lexjansen** [comment]
When talking about Latin-1, it is important to distinguish between ISO Latin-1 and Windows Latin-1. 
ISO Latin-1, also known as ISO 8859-1, is a subset (first 256 characters) of UTF-8. Windows-1252, commonly known as Windows Latin 1, is not a subset of UTF-8.
This page on [stackoverflow](https://stackoverflow.com/questions/19109899/what-is-the-exact-difference-between-windows-1252-and-iso-8859-1) gives a good overview of the differences.

> Windows-1252 is commonly known as Windows Latin 1 or as Windows West European or something like that. It differs from ISO Latin 1, also known as ISO-8859-1 as a character encoding, so that the code range 0x80 to 0x9F is reserved for control characters in ISO-8859-1 (so-called C1 Controls), whereas in Windows-1252, some of the codes there are assigned to printable characters (mostly punctuation characters), others are left undefined.
